core/src/test/java/com/graphhopper/routing/util/EncodingManagerTest.java

```diff
@@ -66,6 +71,13 @@ public void testEncoderAcceptNoException()
     }
 
     @Test
+    public void testEncoderWithWrongVersionIsRejected()
+    {
+        thrown.expect(IllegalArgumentException.class);
```

Comment:
Also there is another way to define expected exception in test
`@Test(expected = IllegalArgumentException.class)`